structuredheadings/styles/numbering.css

```diff
@@ -178,3 +178,20 @@ body {
 .autonumber.autonumber-5.autonumber-r:before {
     content: counter(autonumber-5, lower-roman) ". ";
 }
+
+/*
+styles for List: 1. 1. 1.
+inspired by these:
+https://stackoverflow.com/questions/4098195/can-ordered-list-produce-result-that-looks-like-1-1-1-2-1-3-instead-of-just-1
+https://stackoverflow.com/questions/10405945/html-ordered-list-1-1-1-2-nested-counters-and-scope-not-working/10405962
+http://jsfiddle.net/qGCUk/4/
+*/
+.document .document_body ol.list-decimal-point, .document .document_body ol.list-decimal-point ol {
```

Comment:
At no other point this this file are we using selectors that include things outside this repo (`.document` and `.document_body` are constructs in PolicyStat, not the editor. 

Is there a reason we are doing so now? Is it required? Can't we just key on `ol.list-decimal-point`?

Reply:
Oh. Right. Copy pasted my older experiments.